content/en/blog/_posts/2022-12-6-cross-namespace-data-sources-alpha.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: blog
+title: "Kubernetes v1.26: Alpha support for cross-namespace storage data sources"
+date: 2022-12-6
```

Comment:
@ttakahashi21 Please change the publication date.

Reply:
```suggestion
date: 2023-01-02
```

Please change the path `content/en/blog/_posts/2022-12-6-cross-namespace-data-sources-alpha.md` to match

Reply:
I also changed to file name of "content/en/blog/_posts/2023-01-02-cross-namespace-data-sources-alpha.md".